embc-app/DataInterfaces/DataInterface.LookupData.cs

```diff
@@ -30,7 +30,9 @@ public async Task<IEnumerable<Community>> GetCommunitiesAsync()
             return (await db.Communities
                 .Include(c => c.RegionalDistrict)
                     .ThenInclude(d => d.Region)
-                .ToArrayAsync()).Select(d => d.ToViewModel());
+                .ToArrayAsync())
```

Comment:
ToArray will stop the query, so you want it before the select but after the orderby